.gitlab/common.yml

```diff
@@ -20,31 +20,74 @@
       fi
       echo "REGISTRY=${REGISTRY}" >> build.env
       echo "HARBOR_USER=${HARBOR_USER}" >> build.env
-      echo "HARBOR_DATAFED_GITLAB_CI_REGISTRY=${HARBOR_DATAFED_GITLAB_CI_REGISTRY}" >> build.env
+      echo "HARBOR_DATAFED_GITLAB_CI_REGISTRY_TOKEN=${HARBOR_DATAFED_GITLAB_CI_REGISTRY_TOKEN}" >> build.env
       sed -i 's/\(HARBOR_USER=.*\)\$/\1$$/g' build.env
   artifacts:
     paths:
       - ${COMPONENT}_image.yml 
     reports:
       dotenv: build.env
 
-
+# NOTE
+#
+# Cases have been observed where an image has been pushed to the image registry but 
+# but the image for some reason is not registered with the Harbor instance. The
+# retry loop is used to retry pushing to the registry to add some resliency to the
+# ci pipeline in the case of poor networking issues while attempting to upload
```

Comment:
Very helpful to know 💯

Reply:
I'm glad you think so despite all the spelling and gramattical errors I'm now seeings :).